### PR TITLE
[OTA-2450] Change endpoints used to cancel pending device updates

### PR DESCRIPTION
**Pull request by houcros**

### CODE REVIEW

**Comment by houcros** [comment]
Hey @piotrcichonmee @piotrsmol could you lend me a hand here? I hope it's understandable what I'm trying to do: change the endpoints used to cancel updates for a device.
I see that `cancelMtuUpdate` is used to cancel in the "queue" tab  of the DDV and `cancelApprovalPendingCampaign` is used to cancel in the "approval pending" tab.
I deployed this code to berlin: `cancelMtuUpdate` works as expected but no request is triggered when I click the cancel button in "approval pending", seems like `cancelMtuUpdate` doesn't get called.
Could you please have a look? Feel free to push a commit on top off this branch or ping me if you find out what's wrong. Thanks!

File: ota-plus-web/app/reactapp/src/stores/DevicesStore.js [hunk 1]
```diff
@@ -542,10 +542,10 @@ export default class DevicesStore {
       );
   }
 
-  cancelApprovalPendingCampaingPerDevice(data) {
+  cancelApprovalPendingCampaingPerDevice(campaignId, deviceId) {
     resetAsync(this.mtuCancelAsync, true);
     return axios
-      .post(API_CANCEL_MULTI_TARGET_UPDATE, data)
+      .delete(`${API_CANCEL_UPDATE_PENDING_APPROVAL}/${campaignId}/devices/${deviceId}`)
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/stores/DevicesStore.js]: At first glance it seems you forgot to import `API_CANCEL_UPDATE_PENDING_APPROVAL` from `../config` 😉

File: ota-plus-web/app/reactapp/src/containers/Device.jsx [hunk 2]
```diff
@@ -44,23 +44,14 @@ class Device extends Component {
     const { stores } = this.props;
     const { devicesStore } = stores;
     const deviceId = devicesStore.device.uuid;
-    const data = {
-      correlationId,
-      device: deviceId,
-    };
-    devicesStore.cancelMtuUpdate(data);
+    devicesStore.cancelMtuUpdate(deviceId);
```
Comment (piotrcichonmee) [inline ota-plus-web/app/reactapp/src/containers/Device.jsx]: Please remove correlationId from input parameters of cancelMtuUpdate function.

File: ota-plus-web/app/reactapp/src/stores/DevicesStore.js (hunk 1 above)
Comment (houcros) [reply ota-plus-web/app/reactapp/src/stores/DevicesStore.js]: Thanks, that was it indeed :)

File: ota-plus-web/app/reactapp/src/containers/Device.jsx (hunk 2 above)
Comment (houcros) [reply ota-plus-web/app/reactapp/src/containers/Device.jsx]: Right! Removed.

**Pull request merged**